app/src/main/java/org/mozilla/fenix/addons/AddonsManagementFragment.kt

```diff
@@ -203,13 +204,16 @@ class AddonsManagementFragment : Fragment(R.layout.fragment_add_ons_management),
             onSuccess = {
                 this@AddonsManagementFragment.view?.let { view ->
                     val rootView = activity?.getRootView() ?: view
-                    showSnackBar(
-                        rootView,
-                        getString(
-                            R.string.mozac_feature_addons_successfully_installed,
-                            it.translatedName
-                        )
+                    FenixSnackbar.make(
+                        view = rootView,
+                        duration = FenixSnackbar.LENGTH_SHORT,
+                        isDisplayedWithBrowserToolbar = false
```

Comment:
Hi @JLRoberts, thanks for this! So your _suggested_ patch could as simple as setting `isDisplayedOnBrowserFragment=false` in [here](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/addons/Extensions.kt#L32).

The reasoning behind attaching `rootView` was so that the snackbar would stay visible when you press back to `BrowserFragment`. So by setting `isDisplayedOnBrowserFragment=false`, would it mess up the alignment for when you go back to `BrowserFragment` after the confirmation Snackbar is shown? I think this is a good start to the underlying padding issue. What do you think?

Reply:
I'll check. I thought that previous patch was for fixing some snackbars not being shown depending on the behavior of the add-on. As it seems some are different. I changed it specifically here because it looks like there are alot of snackbars that use the extension method and some need the padding. Some don't. So may need a separate one for showing without padding or just using the actual methods if more crop up to need no padding.